bin.src/makeGainImages.py

```diff
@@ -24,7 +24,7 @@
 import argparse
 
 from lsst.obs.lsst.phosim import PhosimMapper
-import lsst.afw.image as afwImage
+import lsst.afw.image as afwimage
```

Comment:
The DM convention always uses `afwImage` so it's going to be confusing to change this to `afwimage`.

Reply:
The problem you are having is that you aren't using the standard DM `setup.cfg` in the project.
https://developer.lsst.io/python/style.html?highlight=setup%20cfg#flake8-configuration-files

Reply:
Without this 'fix', the lint test fails. Would you prefer if I change it back to camelCase?

======================================== FAILURES ========================================
__________________ FLAKE8-check(ignoring E133 E226 E228 N802 N803 N806) __________________
/home/dthomas/Code/phosim_utils/bin/makeGainImages.py:27:2: N812 lowercase 'lsst.afw.image' imported as non lowercase 'afwImage'

__________________ FLAKE8-check(ignoring E133 E226 E228 N802 N803 N806) __________________
/home/dthomas/Code/phosim_utils/bin.src/makeGainImages.py:27:2: N812 lowercase 'lsst.afw.image' imported as non lowercase 'afwImage'

 generated xml file: /home/dthomas/Code/phosim_utils/tests/.tests/pytest-phosim_utils.xml

----------- coverage: platform linux, python 3.6.6-final-0 -----------
Name                    Stmts   Miss Branch BrPart  Cover
---------------------------------------------------------
tests/test_minimal.py      11      2      2      1    77%
Coverage HTML written to dir tests/.tests/pytest-phosim_utils.xml-htmlcov
Coverage XML written to file tests/.tests/pytest-phosim_utils.xml-cov-phosim_utils.xml

===================== 2 failed, 3 passed, 8 skipped in 1.28 seconds ======================
Global pytest run: failed with 1

Reply:
Great, replaced the afwImage commit with one updating setup.cfg.